src/scss/components/banner/_banner-quote-text.scss

```
@@ -0,0 +1,86 @@
+@charset 'UTF-8';
+
+// Top banner styles
+.irsr-wrapper--banner-quote-text {
+  .su-hero {
+    display: block;
+    min-height: 220px;
+  
+    .su-hero__media {
+      min-height: auto;
+      position: absolute;
+  
+      img {
+        opacity: 0.4;
+      }
+    }
+  
+    .su-hero__card {
+      background: transparent;
+      border: 0;
+      box-shadow: none;
+      left: auto;
+      text-align: center;
+      max-width: unset;
+      margin: 0;
+      bottom: unset;
+  
+      .su-card__contents {
+        @include responsive-spacing('padding-top', 4);
+        @include responsive-spacing('padding-bottom', 5);
+
+        color: $su-color-white;
+        width: 75%;
```

```suggestion
        max-width: 100ch;
```
Let's using a `ch` to match the width limit we have on WYSIWG